saleor/graphql/account/mutations/permission_group.py

```diff
@@ -107,118 +155,106 @@ class Meta:
         description = "Update permission group."
         model = auth_models.Group
         permissions = (AccountPermissions.MANAGE_STAFF,)
-        error_type_class = BulkAccountError
-        error_type_field = "bulk_account_errors"
+        error_type_class = PermissionGroupError
+        error_type_field = "permission_group_errors"
 
+    @classmethod
+    @transaction.atomic
+    def _save_m2m(cls, info, instance, cleaned_data):
+        cls.update_group_permissions_and_users(instance, cleaned_data)
 
-class PermissionGroupDelete(ModelDeleteMutation):
-    class Arguments:
-        id = graphene.ID(description="ID of the group to delete.", required=True)
-
-    class Meta:
-        description = "Delete permission group."
-        model = auth_models.Group
-        permissions = (AccountPermissions.MANAGE_STAFF,)
-        error_type_class = AccountError
-        error_type_field = "account_errors"
+    @classmethod
+    def update_group_permissions_and_users(
+        cls, group: auth_models.Group, cleaned_input: dict
+    ):
+        if "add_users" in cleaned_input:
+            group.user_set.add(*cleaned_input["add_users"])
+        if "remove_users" in cleaned_input:
+            group.user_set.remove(*cleaned_input["remove_users"])
+
+        if "add_permissions" in cleaned_input:
+            group.permissions.add(*cleaned_input["add_permissions"])
+        if "remove_perissions" in cleaned_input:
+            remove_permissions = get_permissions(cleaned_input["remove_perissions"])
```

Comment:
`get_permissions` is needed here? I'm asking because it's not used for adding.

Reply:
It's also used for adding but in `clean_permissions`. But maybe it probably shouldn't be there, because it's not clear what going on in `update_group_permissions_and_users`.

Reply:
I left `get_permission` in `clean_permissions` but moved the one from `update_group_permissions_and_users`  to `clean_input` in `PermissionGrpupUpdate`. I hope it's more clear right now.